package/ingenic-audiodaemon/files/S96iad

```diff
@@ -4,13 +4,20 @@
 
 DAEMON_ARGS="-r -c /etc/iad.json"
 
+audiostreamer_enable="$(get audiolistener)"
+
 start() {
 	starting
 	start_daemon
+
+	# Run listener in background for playing audio over the network
+	[ "$audiostreamer_enable" == "true" ] && nc -ll -p 8081 -e iac -s &
 }
 
 stop() {
 	stopping
+	is_streamer_disabled && quit "Streamer disabled"
+	[ "$audiostreamer_enable" == "true" ] && killall nc
```

Comment:
this will kill any `nc` which is not an option. imp-control uses `nc`, too.

Reply:
@jayfan0 maybe start it using start-stop-daemon and assign it a pid? and then stop it via the same way